nntrainer/layers/acti_func.h

```diff
@@ -588,6 +618,8 @@ class ActiFunc {
   }
 
 private:
+  constexpr static inline float ALPHA = 1.0f; /**< alpha for elu */
```

Comment:
agree. one way that comes to my mind is to have alpha as a parameter.
```
template <typename T = float> static T elu(T x, float alpha = 1.0) {
  return x >= static_cast<T>(0.0) ? x : static_cast<T>(alpha) * (exp(x) - 1);
}
```
in this case, a new `setActivation` function needs to be implemented.

Reply:
@skykongkong8 Thank you for comment. Because `ALPHA` in used in both `elu` and `eluPrime`, I thought it is better to store it in the class, not in method level. And I thought constant should be written in capital letter, like `NEGATIVE_SLOPE` :sweat:. Should we convert both `ALPHA` and `NEGATIVE_SLOPE` to lower case? Cause I'm pretty newcomer to both NNTrainer and C++, I'll follow you (and your repository)'s convention.

Reply:
@djeong20 Convert alpha as a parameter could be an option. Thank you for comment. But in my opinion, it should be handle in another PR in consideration of separation of concern. What do you think?

Reply:
you're right. let's make changes in a later PR for `ALPHA`, `BETA` (from #2545), and `NEGATIVE_SLOPE`!

Reply:
@heka1024  Sounds you have a point. However, still cannot agree with upper-case usage.

1. According to [here](https://lefticus.gitbooks.io/cpp-best-practices/content/03-Style.html) and [here](https://www.geeksforgeeks.org/naming-convention-in-c/), yes it is recommended to use all-upper-case for constants, but isn't it for "constant variables" like `PI` or `INT_MAX`? As far as I am concerned, `alpha` is more like a scalar to determine the effect of the function, not really a mathematical nor system-defined constant
2. There is no such convention in the NNTrainer to use all-upper-case naming except for the macros I guess.

Reply:
@skykongkong8 Thank you for kind guidance. 
> alpha is more like a scalar to determine the effect of the function, neither of mathematical nor system-defined constant

I understood this, and just changed it to lower case.